cmd/minikube/cmd/start.go

```diff
@@ -113,6 +113,12 @@ func runStart(cmd *cobra.Command, args []string) {
 	libMachineClient := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 	defer libMachineClient.Close()
 
+	_, err := shellCfgUnset(libMachineClient)
+	if err != nil {
+		glog.Errorln("Error unsetting existing docker environment variables:", err)
+                os.Exit(1)
```

Comment:
@LalatenduMohanty better to do in a single line so you don't have to change below anything.

```
if _, err := shellCfgUnset(libMachineClient); err != nil {
        glog.Errorln("Error unsetting existing docker environment variables:", err)
        os.Exit(1)
}
```